cmd/ci-operator-configresolver/main.go

```diff
@@ -128,6 +130,13 @@ func validateOptions(o options) error {
 			return fmt.Errorf("--registry points to a nonexistent directory: %v", err)
 		}
 	}
+	if o.prowPath == "" {
+		return fmt.Errorf("--prow-config is required")
+	} else {
+		if _, err := os.Stat(o.prowPath); err != nil && os.IsNotExist(err) {
+			return fmt.Errorf("--prow-config points to a nonexistent file/directory: %v", err)
```

Comment:
Why don't we care to show off any other types of errors?

Reply:
That's how the `--registry` and `--config` flags work as well. I can change them to print other errors as well.